tox.ini

```diff
@@ -981,23 +984,29 @@ commands =
 description = Deploy Syft to a local Kubernetes cluster with Devspace
 changedir = {toxinidir}/packages/grid
 passenv=HOME, USER
+setenv=
+    KUBE_CONTEXT = {env:KUBE_CONTEXT:k3d-syft-dev}
+    DEVSPACE_PROFILE = {env:DEVSPACE_PROFILE:}
```

Comment:
For launching gateway it could be like
```
CLUSTER_NAME="syft-gateway" NODE_PORT=9081 tox -e dev.k8s.start
KUBE_CONTEXT="k3d-syft-gateway" DEVSPACE_PROFILE="-p gateway" tox -e dev.k8s.hotreload
```

Reply:
Since kubecontext is basically `k3d-$CLUSTER_NAME`, it's redundant. I'll push an update for this.